en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md

```diff
@@ -200,6 +200,9 @@ Follow the  instructions below to change the type of the default datasources.
         ``` java
         driver="com.mysql.cj.jdbc.Driver"
         ```
+    
+    !!! note "Recommendation"
```

Comment:
@vithu30 , this note needs to go into the 3.1.0 docs as well.

Reply:
Added to 3.1.0 with PR - https://github.com/wso2/docs-apim/pull/2074